Sources/HeliumLogger/HeliumLogger.swift

```diff
@@ -110,53 +227,74 @@ extension HeliumLogger : Logger {
     public func log(_ type: LoggerMessageType, msg: String,
         functionName: String, lineNum: Int, fileName: String ) {
 
-            let color : TerminalColor
+        guard type.rawValue >= self.type.rawValue else {
+            return
+        }
 
-            switch type {
-                case .warning:
-                    color = .yellow
-                case .error:
-                    color = .red
-                default:
-                    color = .foreground
-            }
+        let date = dateFormatter.string(from: Date())
```

Comment:
Doesn't this do the work of retrieving the `Date` even if we're not logging dates?